CODEOWNERS

```diff
@@ -31,7 +31,19 @@
 /vault/plugin_catalog.go      @hashicorp/vault-ecosystem
 
 # Content on developer.hashicorp.com
-/website/ @hashicorp/vault-education-approvers
+
+# Engineering and web presence get notified of, and can approve changes to web tooling, but not content.
+
+/website/               @hashicorp/web-presence @hashicorp/vault
```

Comment:
Why are we adding `@hashicorp/vault` as an owner for files under `/website/` and the content-specific folders below?

Reply:
My reasoning was that we would want the engineering team to have complete control over their own repo, in case (and I know this is very unlikely) there was a problem in one of these files that blocked a release or some other emergency. I don't think we explicitly exclude these directories from CI and release tooling, so I just want to make sure there's nothing in here that engineering couldn't fix on their own in a pinch.